qkeras/qlayers.py

```diff
@@ -1973,3 +2108,39 @@ def model_quantize(model,
         qlayer.set_weights(copy.deepcopy(layer.get_weights()))
 
   return qmodel, custom_objects
+
+
+def quantized_model_from_json(json_string, custom_objects=None):
+  if not custom_objects:
+    custom_objects = {}
+
+  # let's make a deep copy to make sure our objects are not shared elsewhere
+  custom_objects = copy.deepcopy(custom_objects)
+
+  custom_objects["QDense"] = QDense
+  custom_objects["QConv1D"] = QConv1D
+  custom_objects["QConv2D"] = QConv2D
+  custom_objects["QDepthwiseConv2D"] = QDepthwiseConv2D
+  custom_objects["QAveragePooling2D"] = QAveragePooling2D
+  custom_objects["QActivation"] = QActivation
+  custom_objects["Clip"] = Clip
+  custom_objects["quantized_bits"] = quantized_bits
+  custom_objects["bernoulli"] = bernoulli
+  custom_objects["stochastic_ternary"] = stochastic_ternary
+  custom_objects["ternary"] = ternary
+  custom_objects["stochastic_binary"] = stochastic_binary
+  custom_objects["binary"] = binary
+  custom_objects["quantized_relu"] = quantized_relu
+  custom_objects["quantized_ulaw"] = quantized_ulaw
+  custom_objects["quantized_tanh"] = quantized_tanh
+  custom_objects["quantized_po2"] = quantized_po2
+  custom_objects["quantized_relu_po2"] = quantized_relu_po2
+
+  from .qnormalization import QBatchNormalization
```

Comment:
could you put this line to the head of this file?

Reply:
`qnormalization` uses stuff from `qlayers` so I cannot import it at the head of the file without making a mess (using `qlayers.Clip` and `qnormalization.QBatchNormalization`). I can move it to the head of that method if that provides more clarity, or we can restructure the modules to avoid this. The latter option may be more suited to a separate PR.

Reply:
The second option is what I was thinking. qlayers.py is too long. Let's create PR to refactor it.